### PR TITLE
Remove repetitive words

### PR DESCRIPTION
**Pull request by cuishuang**

### CODE REVIEW

File: credential/cargo-credential/examples/stdout-redirected.rs [hunk 1]
```diff
@@ -15,7 +15,7 @@ impl Credential for MyCredential {
         _args: &[&str],
     ) -> Result<CredentialResponse, Error> {
         // Informational messages should be sent on stderr.
-        eprintln!("message on stderr should be sent the the parent process");
+        eprintln!("message on stderr should be sent the parent process");
```
Comment (ehuss) [inline credential/cargo-credential/examples/stdout-redirected.rs:18]: I believe this is intended to be "to the parent process".

```suggestion
        eprintln!("message on stderr should be sent to the parent process");
```

File: credential/cargo-credential/tests/examples.rs [hunk 2]
```diff
@@ -15,7 +15,7 @@ fn stdout_redirected() {
         .arg("--cargo-plugin")
         .assert()
         .stdout_eq(format!("{hello}\n{err_not_supported}\n"))
-        .stderr_eq("message on stderr should be sent the the parent process\n")
+        .stderr_eq("message on stderr should be sent the parent process\n")
```
Comment (ehuss) [inline credential/cargo-credential/tests/examples.rs:18]: Same here.

```suggestion
        .stderr_eq("message on stderr should be sent to the parent process\n")
```

File: credential/cargo-credential/examples/stdout-redirected.rs (hunk 1 above)
Comment (cuishuang) [reply credential/cargo-credential/examples/stdout-redirected.rs]: Thanks. Done

File: credential/cargo-credential/tests/examples.rs (hunk 2 above)
Comment (cuishuang) [reply credential/cargo-credential/tests/examples.rs]: Thanks.  Done.

**Pull request merged**